### PR TITLE
Add show page to photo

### PR DESCRIPTION
**Pull request by sawleha16**
1.Added show page to photo 
Screenshot from 2022-12-20 17-52-59
2. Added comments on photo 
Screenshot from 2022-12-20 17-52-49

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,29 @@
  *= require_tree .
  *= require_self
  */
+
+ <style>
```
Comment (ahmad19) [inline app/assets/stylesheets/application.css]: we do not need style attribute inside the css file.

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,29 @@
...
+  color: red;
+  background-color: transparent;
+  text-decoration: underline;
+}
+
+a:active {
+  color: yellow;
+  background-color: transparent;
+  text-decoration: underline;
+}
```
Comment (ahmad19) [inline app/assets/stylesheets/application.css:40]: where does this CSS actually implied ?
Can you add a screenshot showing the result of this CSS ?

File: test/fixtures/comments.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (ahmad19) [inline test/fixtures/comments.yml]: remove this file.

File: test/models/comment_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (ahmad19) [inline test/models/comment_test.rb]: remove this file.

**Comment by ahmad19** [comment]
i have noticed that all albums will be created visible to every user. Although it should not happen. lets fix this behaviour in the next ticket.

File: app/assets/stylesheets/application.css (hunk 2 above)
Comment (sawleha16) [reply app/assets/stylesheets/application.css:40]: 1.Here
Screenshot from 2022-12-21 18-34-52

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (sawleha16) [reply app/assets/stylesheets/application.css]: Removed

File: test/models/comment_test.rb (hunk 4 above)
Comment (sawleha16) [reply test/models/comment_test.rb]: Removed

File: test/fixtures/comments.yml (hunk 3 above)
Comment (sawleha16) [reply test/fixtures/comments.yml]: Removed

**Review by ahmad19 — APPROVED** [review]

**Pull request merged**